### PR TITLE
Adjust CPU resources

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
We are experiencing timeouts for things that should normally not time out. Like TLS handshakes and node-exporter scrapes. It turns out that very low CPU limits actually work and really limit the CPU time we can spend. Who could have expected that?!

This PR increases the CPU shares for most components and allows more bursting. Likewise, this PR adds resource constraints for Thanos based on our current cloud-eu cluster.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: config/monitoring/prometheus/values.yaml [hunk 1]
```diff
@@ -213,13 +213,37 @@ prometheus:
...
+          memory: 32Mi
+        limits:
+          cpu: 300m
+          memory: 1Gi
     thanosStore:
-      resources: {}
+      resources:
+        requests:
+          cpu: 100m
+          memory: 1536Mi
```
Comment (kron4eg) [inline config/monitoring/prometheus/values.yaml:227]: What a strange number. Where does it come from?

File: config/monitoring/prometheus/values.yaml [hunk 2]
```diff
@@ -213,13 +213,37 @@ prometheus:
...
+        limits:
+          cpu: 250m
+          memory: 3Gi
     thanosQuery:
-      resources: {}
+      resources:
+        requests:
+          cpu: 500m
```
Comment (kron4eg) [inline config/monitoring/prometheus/values.yaml]: Just to start 50% of vCPU?!

**Review by kron4eg — COMMENTED** [review]
left couple of questions

File: config/monitoring/prometheus/values.yaml (hunk 1 above)
Comment (xrstf) [reply config/monitoring/prometheus/values.yaml:227]: 3 * 512Mi

**Review by kron4eg — APPROVED** [review]
/approve no-issue

**Pull request merged**